charts/loadtester/templates/deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
         app.kubernetes.io/name: {{ include "loadtester.name" . }}
       annotations:
         appmesh.k8s.aws/ports: "444"
+        # TODO check with Stefan if including the required inbound port exclusion for OSM is okay here
+        # TODO check with Stefan the necessary ports for flagger-loadtester
+        openservicemesh.io/inbound-port-exclusion-list: "80, 8080"
```

Comment:
If OSM behaves like AppMesh and blocks traffic by default, then yes we need to exclude the ports.

Reply:
Yes OSM blocks traffic by default (restrictive mtls by default), and these are required. Thanks for giving permission to leave these here.

Reply:
Removed todo comments about this through 22789727

Reply:
Added osm inbound port exclusion list to flagger-loadtester. This is so that the flagger-controller can communicate with the flagger-loadtester.